.changeset/eighty-insects-tickle.md

```diff
@@ -0,0 +1,130 @@
+---
+'wagmi': minor
+---
+
+**Breaking**: With the introduction of the [`wagmi/chains` entrypoint](/react/chains#wagmichains), `wagmi` no longer exports the following:
```

Comment:
Technically the `wagmi/chains` entrypoint has already been around, but nbd to spotlight it here.